### DISCUSSION TITLE
using goto without http

### DISCUSSION THREAD

**Discussion by KarianneM** [getgauge/taiko#1786] [General]
Hello,

I have another question similar to https://github.com/getgauge/taiko/issues/903.

Is it possible to skip the fallback to http while using goto? we would like to use:

goto("chrome://settings/content/pdfDocuments")

Regards,
Karianne

**Comment by zabil** [comment]
This is fixed in the latest version of Taiko 1.1.2 Please refer https://github.com/getgauge/taiko/issues/1720

**Reply by KarianneM** [reply]
Thanks Zabil, it does work with version 1.1.2 of Taiko only now I run into the problem of https://github.com/getgauge/taiko/issues/1777

**Comment by KarianneM** [comment]
When I switch to headless the URL fails

Error: Navigation to url chrome://settings/content/pdfDocuments failed. REASON: net::ERR_INVALID_URL

is this covered in 1.1.4?

**Reply by zabil** [reply]
I see this error and googled up to find that the settings url is not permitted in headless mode 
https://github.com/go-rod/rod/issues/217

You can try using the https://docs.taiko.dev/api/client/ api and use chrome devtools protocol to read values instead.

**Reply by KarianneM** [reply]
thx I'll use the headfull for now
